.pa11yci.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+	"defaults": {
+		"timeout": 1000,
+		"page": {
+			"viewport": {
+				"width": 320,
```

Comment:
Is it possible to run the test twice on different widths - some features are displayed very differently depending on the screen size